src/Symfony/Component/Lock/SharedLockStoreInterface.php

```diff
@@ -22,15 +21,7 @@ interface SharedLockStoreInterface extends PersistingStoreInterface
     /**
      * Stores the resource if it's not locked for reading by someone else.
      *
-     * @throws NotSupportedException
```

Comment:
Is it a BC-break?

Reply:
I don't think so

Reply:
nvm, this interface has been introduced in 5.2